src/targets/gpu/fuse_ops.cpp

```diff
@@ -737,6 +788,7 @@ struct find_conv_bias_relu
 
 void fuse_ops::apply(program& p) const
 {
+    match::find_matches(p, find_layernorm{});
```

Comment:
Can this be put in the `find_matches` call below?

Reply:
moved